<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>manager_operation</class>
 <widget class="QMainWindow" name="manager_operation">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>955</width>
    <height>708</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>微软雅黑</family>
    <pointsize>12</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>超市 管理系统后台</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>-10</y>
      <width>951</width>
      <height>671</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <spacer name="verticalSpacer_2">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Fixed</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeType">
          <enum>QSizePolicy::Minimum</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QTabWidget" name="tabWidget">
         <property name="whatsThis">
          <string/>
         </property>
         <property name="accessibleName">
          <string/>
         </property>
         <property name="styleSheet">
          <string notr="true"/>
         </property>
         <property name="currentIndex">
          <number>0</number>
         </property>
         <widget class="QWidget" name="staff_tab">
          <property name="whatsThis">
           <string/>
          </property>
          <property name="layoutDirection">
           <enum>Qt::LeftToRight</enum>
          </property>
          <attribute name="title">
           <string>员工管理</string>
          </attribute>
          <widget class="QWidget" name="horizontalLayoutWidget">
           <property name="geometry">
            <rect>
             <x>40</x>
             <y>490</y>
             <width>821</width>
             <height>51</height>
            </rect>
           </property>
           <layout class="QHBoxLayout" name="horizontalLayout">
            <item>
             <widget class="QPushButton" name="add_staff_Btn">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="text">
               <string>添加员工</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="delete__staff_Btn">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="text">
               <string>   删除员工</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="return_Btn">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="text">
               <string>返回</string>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
          <widget class="QTableView" name="staff_tableView">
           <property name="geometry">
            <rect>
             <x>40</x>
             <y>30</y>
             <width>821</width>
             <height>441</height>
            </rect>
           </property>
           <property name="editTriggers">
            <set>QAbstractItemView::NoEditTriggers</set>
           </property>
          </widget>
         </widget>
         <widget class="QWidget" name="goods_tab">
          <attribute name="title">
           <string>货物管理</string>
          </attribute>
          <widget class="QTableView" name="goods_tableView">
           <property name="geometry">
            <rect>
             <x>40</x>
             <y>30</y>
             <width>811</width>
             <height>461</height>
            </rect>
           </property>
           <property name="editTriggers">
            <set>QAbstractItemView::NoEditTriggers</set>
           </property>
          </widget>
          <widget class="QWidget" name="horizontalLayoutWidget_2">
           <property name="geometry">
            <rect>
             <x>340</x>
             <y>500</y>
             <width>509</width>
             <height>51</height>
            </rect>
           </property>
           <layout class="QHBoxLayout" name="horizontalLayout_2">
            <item>
             <widget class="QPushButton" name="pushButton">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="text">
               <string>进货</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="add_goods_Btn">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="text">
               <string> 新增商品</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="return_Btn_2">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="text">
               <string> 返回</string>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </widget>
         <widget class="QWidget" name="sales_record_tab">
          <attribute name="title">
           <string>销售记录</string>
          </attribute>
          <widget class="QTextBrowser" name="sales_record_text">
           <property name="geometry">
            <rect>
             <x>40</x>
             <y>20</y>
             <width>341</width>
             <height>481</height>
            </rect>
           </property>
          </widget>
          <widget class="QPushButton" name="Btn">
           <property name="geometry">
            <rect>
             <x>260</x>
             <y>520</y>
             <width>111</width>
             <height>51</height>
            </rect>
           </property>
           <property name="text">
            <string> 返回</string>
           </property>
          </widget>
         </widget>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer_2">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeType">
          <enum>QSizePolicy::Fixed</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
     <item>
      <spacer name="verticalSpacer">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Minimum</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>955</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
